README.md

```diff
@@ -49,15 +50,21 @@ A curated list of open source projects used in nuclear science and engineering.
 
 - [Nek5000](https://github.com/Nek5000/Nek5000) — Spectral-element CFD code
 - [OpenFOAM](https://www.openfoam.com/) — Finite volume CFD code
+- [nekRS](https://github.com/Nek5000/nekRS) — MPI+X hybrid parallelism Spectral-element CFD code supporting CPU, CUDA, HIP and OpenCL
 
 ## Multiphysics
 
 - [MOOSE](https://github.com/idaholab/moose) — Finite-element, multiphysics framework
 - [Okapi](https://github.com/aprilnovak/okapi) — OpenMC wrapped as a MOOSE app
 - [ENRICO](https://github.com/enrico-dev/enrico) — Monte Carlo + CFD coupling application
 
+## Molten Salt Reactor
+
+- [moltres](https://github.com/arfc/moltres) — A molten salt reactor simulator code
+
 ## Other
 
 - [PyNE](https://github.com/pyne/pyne) — Python/C++ nuclear engineering toolkit
 - [ARMI](https://github.com/terrapower/armi) — Reactor analysis automation framework
+- [ARFC](https://github.com/arfc) — A research group focused on modeling and simulation of advanced nuclear reactors and fuel cycles
```

Comment:
Maybe we should add a section for research groups that are heavily invested in open source tools.  This category doesn't quite seem like the right place for ARFC.

Reply:
Thanks for the suggestion @gonuke.

Reply:
Is it ok now @gonuke?